projects/iio_demo/src/app/main.c

```diff
@@ -45,7 +45,7 @@
 #include "iio_demo_dev.h"
 #include "app_config.h"
 #include "parameters.h"
-#include "error.h"
+#include <errno.h>
```

Comment:
You need to replace FAILURE with -1 in platform_init

Reply:
fixed in latest version